src/pages/docs/administration/copilot/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-01-01
+modDate: 2023-10-04
+title: Octopus Copilot integration
+description: How to use the Octopus Copilot integration
+navOrder: 100
+hideInThisSection: true
+---
+
+The Octopus Copilot integration allows read only queries of cloud Octopus instances via GitHub Copilot.
+
+:::div{.warning}
+Octopus Copilot is an experiment. It is not covered by service level agreements.
+:::
+
+The goal of Octopus Copilot is to provide a middle ground between the web based UI and the REST API. Prior to Octopus Copilot, tasks that can not be performed via the web UI could only be achieved through custom scripts that interacted with the REST API. Working with the REST API requires a good understanding of the Octopus domain model and the ability to write custom code. Octopus Copilot provides the ability to query the configuration and state of an Octopus instance via natural language prompts.
+
+## Getting started
+
+Octopus Copilot has 3 prerequisites:
+
+* An Octopus Cloud instance
+* An API key
+* A GitHub Copilot account
+
+### Creating an Octopus cloud instance
+Octopus Copilot is available for cloud Octopus instances. Click [here](https://octopus.com/start) to sign up for a trial cloud Octopus instance.
+
+### Creating the Octopus API key
+Octopus Copilot requires an API key to interact with the Octopus server. 
+
+Octopus Copilot only requires read access to the Octopus instance. It is recommended that you create a service account that belongs to a team with read-only permissions. The documentation [here](https://github.com/OctopusSolutionsEngineering/OctopusCopilot?tab=readme-ov-file#creating-a-service-account) provides a sample Terraform module to create a read-only role, a team referencing the role, and a service account belonging to the team.
```

Comment:
I would link to the docs page for service accounts too. Bit simpler for teams not using the terraform provider.

Reply:
:+1: Good idea